scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -139,20 +163,24 @@ trait AndroidAppModule extends JavaModule {
     (jarFiles, resFolders)
   }
 
+  def res: T[Seq[PathRef]] = Task.Sources { millSourcePath / "res" }
```

Comment:
imo this can be confusing for the end-user to have both `res` and `resources` in the build script API layout. Having just `resources` imo is fine, it doesn't cause issues.

Reply:
there are a few issues that are mixing the resources and I'll look into untangling them on a subsequent PR, for now I separated them to make it easier to build everything , if it's ok with you

Reply:
The maybe it is worth to make this one `private` for now if there is a chance it goes away at some point?

Reply:
the test modules need to override to add there own res directories but I suppose they can be private nevertheless. let me give it a go.

also would be good to chat together in general if you are working in this as well to get in sync, please drop me an email if you'd like :)

Reply:
It can be `protected` then to support the override. Can you please explain the issue that leads to the need of this method?

Reply:
wait, yeah I managed to get rid of it, thanks!